City-of-Helsinki/drupal-hdbt
# UHF-147 Basic site features

How to test:
- Check PR from https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/2
